pytorch_blade/.bazelrc

```diff
@@ -3,6 +3,7 @@ try-import ../tf_community/.bazelrc.user
 
 build --disk_cache=~/.cache
 build:torch_cuda --config=cuda --config=disc_cuda --define enable_cuda=true
-build:torch_cpu --config=disc_cpu --config=disc_mkldnn
+build:torch_x86 --config=disc_x86 --config=disc_mkldnn
```

Comment:
what if it's targeting arm CPU with a cuda GPU like jetson? Should both `torch_cuda` and `torch_aarch64` be specified?  While for current situations, we just specify `torch_cuda` for X86 CPU with a cuda GPU.

Or, we just will never do cross compiling? There flags are for accelerators only, host code depends on compiling platform.